medusa/clients/torrent/qbittorrent.py

```diff
@@ -36,30 +36,73 @@ def __init__(self, host=None, username=None, password=None):
     @property
     def api(self):
         """Get API version."""
+        # Update the auth method to v2
+        self._get_auth = self._get_auth_v2
+        # Attempt to get API v2 version first
+        self.url = '{host}api/v2/app/webapiVersion'.format(host=self.host)
+        try:
+            version = self.session.get(self.url, verify=app.TORRENT_VERIFY_CERT,
+                                       cookies=self.session.cookies)
+            # Make sure version is using the (major, minor, release) format
+            version = map(int, version.content.split('.'))
```

Comment:
I should have:
`from six.moves import map`
and
```suggestion
            version = list(map(int, version.text.split('.')))
```
----
or
```suggestion
            version = [int(v) for v in version.text.split('.')]
```